### PR TITLE
Add USR2 signal to reread the config

### PR DESCRIPTION
**Pull request by LIQRGV**
Short Description: 
Usage: kill -USR2 <pokecli PID>

Now we can send USR2 signal instead of restart the program

### CODE REVIEW

**Comment by douglascamata** [comment]
LGTM

File: pokecli.py [hunk 1]
```diff
@@ -206,15 +211,20 @@ def main():
...
+
     except KeyboardInterrupt:
         logger.log('[x] Exiting PokemonGo Bot', 'red')
         # TODO Add number of pokemon catched, pokestops visited, highest CP
         # pokemon catched, etc.
 
+signal.signal(signal.SIGUSR2, reread_config)
 
 if __name__ == '__main__':
-    main()
```
Comment (solderzzc) [inline pokecli.py]: why we need a where here?
so are you working on a external signal control?

**Comment by Zgrkaralar** [comment]
But my suggestion getting that signal name from config.json. Beucase i have 20 working bot if i use that when i send this signal it's will kill all @douglascamata @solderzzc

**Comment by douglascamata** [comment]
@Zgrkaralar but why would you send that SIGNAL to all processes instead of just the process you want to reload the config?

File: pokecli.py [hunk 2]
```diff
@@ -206,15 +211,20 @@ def main():
...
+    while True:
```
Comment (douglascamata) [inline pokecli.py]: This is indeed weird. Please try to keep all necessary logic inside the `main` function (including loop) and remove this from the ` if __name__ == '__main__'` block. Make it as simple as possible, please.

**Comment by LIQRGV** [comment]
Ah ya, the logic should implemented on main function.
Thanks for that suggestion.

Also add the code on `if __name__ == '__main__'` caused the application run forever.
My bad.

**Comment by surfaace** [comment]
@LIQRGV please update files accordingly, also recreate PR to merge into dev branch

**Pull request closed without merging**